core/src/test/kotlin/repository/UserRepositoryIT.kt

```diff
@@ -0,0 +1,176 @@
+package repository
+
+import org.bson.types.ObjectId
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import reactor.test.StepVerifier
+import rys.ajaxpetproject.model.MongoUser
+import rys.ajaxpetproject.repository.UserRepository
+
+@DbIntegrationTest
+class UserRepositoryIT {
+    @Autowired
+    private lateinit var userRepository: UserRepository
+
+    @BeforeEach
+    fun init() {
+        userRepository.deleteAll().block()
+    }
+
+    @Test
+    fun `should return user when findById() success`() {
+        //GIVEN
+        val id = ObjectId("652f9203deb9044f309a7dd9")
+        val user = MongoUser(
+            id = id, userName = "Successful_findUserById - ${System.nanoTime()}",
+            password = "password"
+        )
+        userRepository.save(user).block()
+
+        //WHEN //THEN
+        StepVerifier.create(userRepository.findById(id))
+            .expectSubscription()
+            .expectNext(user)
+            .verifyComplete()
+    }
+
+    @Test
+    fun `should return user when findByName() success`() {
```

Comment:
```suggestion
    fun `should return user by name when user with provided name is exist`() {
```

Reply:
Do I need to specify that return is actually are Mono ? 
I mean : 
"should return `mono of user` by name when user with provided name is exist"
instead of 
"should return `user` by name when user with provided name is exist"

Reply:
@ajax-gnatiuk-s

Reply:
mentioning of mono or flux is optional. I prefer avoiding these two words